Sources/ToucanModels/Query/Content+RunQuery.swift

```diff
@@ -18,7 +18,15 @@ public extension [Content] {
         let contents = self.filter {
             query.contentType == $0.definition.id
         }
-        return filter(contents: contents, using: query)
+        let now = Date().timeIntervalSince1970
```

Comment:
I believe we should use the generation date here, otherwise different queries might use different generation times, which could lead to misunderstandings/bugs.